stock_picking_value_triple_discount/models/stock_pack_operation.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class StockPackOperation(models.Model):
+    _inherit = "stock.pack.operation"
+
+    sale_discount2 = fields.Float(
+        compute='_compute_sale_order_line_fields',
+        digits=dp.get_precision('Discount'),
+        string="Sale discount 2 (%)")
+    sale_discount3 = fields.Float(
+        compute='_compute_sale_order_line_fields',
+        digits=dp.get_precision('Discount'),
+        string="Sale discount 3 (%)")
+
+    @api.multi
+    def sale_lines_values(self, sale_lines):
+        discount_extra = (sale_lines.mapped('discount2') +
```

Comment:
Why do you need to compute again all the fields? Please make the other module inheritable using some hook methods for avoiding rewriting entire method.